src/discord/events/message/registerTeam.ts

```diff
@@ -121,9 +121,10 @@ export const regSubCommands: SubCommands = {
           ],
         },
       });
-      ctx.currentCommand = undefined;
-      ctx.nextStep = undefined;
+      // ctx.currentCommand = undefined;
+      // ctx.nextStep = undefined;
```

Comment:
So this was the issue? Can you delete both of these comments? Do the tests still pass?

Reply:
Yes, after registering the team the bot no longer thinks the next message is the team room. The registerTeam.tests.ts does not pass because it checks if those properties are undefined. Is this alright?